kayenta-judge/src/test/scala/com/netflix/kayenta/judge/ScorerSuite.scala

```diff
@@ -98,7 +97,7 @@ class ScorerSuite extends FunSuite {
 
     assert(scores.groupScores.isDefined)
     val groupScores = scores.groupScores.get
-    assert(groupScores.size == 2)
+    assert(groupScores.lengthCompare(2) == 0)
```

Comment:
This change seems less readable.  Was it made for a specific problem or was this an IDE suggestion?

Reply:
This was an IDE suggestion.